ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Block.hs

```diff
@@ -79,6 +79,7 @@ instance Condense (ShelleyHash c) where
   condense = show . unShelleyHash
 
 instance Crypto c => ConvertRawHash (ShelleyBlock c) where
+  -- TODO use 'ShortByteString' for 'Hash'
```

Comment:
We should switch the [`Hash` type in `cardano-ledger-specs`](https://github.com/input-output-hk/cardano-base/blob/c388d765566ad5e6141c586425bcd7b4efbfec8a/cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs#L68) to use `ShortByteString`.

Reply:
Done in https://github.com/input-output-hk/cardano-base/pull/144